py_bind/optv/parameters.pyx

```diff
@@ -0,0 +1,58 @@
+#Implementation of Python binding to parameters.h
+from libc.stdlib cimport malloc, free
+
+cdef class Py_mm_np:
```

Comment:
Let's call it MultimediaParams so future users will not be as confused as we are about what each thing does on the C side.